pkgs/by-name/ta/tayga/package.nix

```diff
@@ -5,15 +5,20 @@
   nixosTests,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   version = "0.9.2";
   pname = "tayga";
 
   src = fetchurl {
-    url = "http://www.litech.org/${pname}/${pname}-${version}.tar.bz2";
+    url = "http://www.litech.org/${finalAttrs.pname}/${finalAttrs.pname}-${finalAttrs.version}.tar.bz2";
```

Comment:
May as well also swap out `pname` in the URL for hardcoding `tayga` since that seems commonly regarded as preferable (seeing as how if you _did_ change the package name, for some downstream derivative, you would _not_ want the sourcecode to be looked for at a different location).

Reply:
sounds good, changed!